src/analyzing/AST_to_IL.ml

```diff
@@ -865,8 +865,7 @@ and expr_lazy_op env op tok arg0 args eorig =
   in
   mk_e (Operator ((op, tok), arg0' :: args')) eorig
 
-and call_generic env ?(void = false) tok e args =
-  let eorig = SameAs (G.Call (e, args) |> G.e) in
```

Comment:
Do we have a test that would fail if we ever undo this change by mistake?

Reply:
Good catch. I remember I had a test, but turns out I don't.

I just added a test for this now.